src/mixins/payment-mixin.ts

```diff
@@ -536,7 +536,7 @@ export class PaymentMixin extends Mixins(ActionMixin) {
       if (!paymentResponse) throw new Error('Got error from getNameRequestPayment()')
 
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      const { payment, sbcPayment = { receipts: [], status_code: '' }, statusCode, completionDate } = paymentResponse
+      const { payment, sbcPayment = { receipts: [], status_code: '' }, statusCode, completionDate } = paymentResponse[0]
```

Comment:
We'll need to discuss this in the meeting, Andriy found this

Reply:
The old and new code probably evaluate the same (thanks, Javascript).

Reply:
Yup, I think we'll probably have to go through all the namex routes and do a side by side to figure out which responses changes to an array or not

Reply:
What I mean is that I think `paymentResponse` and `paymentResponse[0]` are equivalent,

Reply:
I had no idea

Reply:
<img width="1047" alt="Screenshot 2024-05-23 at 3 27 19 PM" src="https://github.com/bcgov/namerequest/assets/22973013/fc21605b-adca-4844-87b6-583906481c13">
<img width="1047" alt="Screenshot 2024-05-23 at 3 27 14 PM" src="https://github.com/bcgov/namerequest/assets/22973013/55d59aff-7d92-415b-8570-41f23c9456f1">
running namerequest locally against test api looks like values are not initalized

Reply:
I'm pretty sure I saw that somewhere. Maybe the TS -> JS cross-compilation does something different.

So that complicates things if different routes give different payment responses...

Reply:
Darn, I could have sworn I saw this behaviour before.

Thanks for the correction, Andrew.

Reply:
Severin, do you agree this means there may be a bug in namerequest right now? Not sure how these variables are used currently

Reply:
Yes. If payment response is an array, and the Namerequest UI code tries to spread the array instead of the first element, that's a bug.